x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -266,24 +297,19 @@ export const NoFindingsStates = ({ postureType }: { postureType: PostureTypes })
       .filter((idxDetails) => idxDetails.status === 'unprivileged')
       .map((idxDetails: IndexDetails) => idxDetails.index)
       .sort((a, b) => a.localeCompare(b));
```

Comment:
I know the fix didn't touch this part, but it bugs me the fact of having to perform this logic, maybe the status API could handle this 🤔